integration-test/build.gradle.kts

```diff
@@ -118,6 +118,8 @@ dependencies {
   }
   testImplementation("org.apache.iceberg:iceberg-spark-runtime-${sparkMajorVersion}_$scalaVersion:$icebergVersion")
   testImplementation("org.apache.kyuubi:kyuubi-spark-connector-hive_$scalaVersion:$kyuubiVersion")
+  testImplementation("org.apache.iceberg:iceberg-core:$icebergVersion")
```

Comment:
why Iceberg REST IT   introduce the two modules while hive backend not?

Reply:
`ObjectMapper` was shaded to `org.apache.iceberg.shaded.com.fasterxml.jackson.databind.ObjectMapper` in `iceberg-spark-runtime-xxx`.
so the Method signature was not `registerAll(Lcom/fasterxml/jackson/databind/ObjectMapper;)V`, but was registerAll(Lorg/apache/iceberg/shaded/com/fasterxml/jackson/databind/ObjectMapper;)V, it causes the Icebergrestservice to fail to start.

```
Caused by: java.util.concurrent.ExecutionException: java.lang.NoSuchMethodError: org.apache.iceberg.rest.RESTSerializers.registerAll(Lcom/fasterxml/jackson/databind/ObjectMapper;)V
	at java.util.concurrent.FutureTask.report(FutureTask.java:122)
	at java.util.concurrent.FutureTask.get(FutureTask.java:206)
	at com.datastrato.gravitino.integration.test.MiniGravitino.start(MiniGravitino.java:159)
	... 76 more
Caused by: java.lang.NoSuchMethodError: org.apache.iceberg.rest.RESTSerializers.registerAll(Lcom/fasterxml/jackson/databind/ObjectMapper;)V
	at com.datastrato.gravitino.catalog.lakehouse.iceberg.web.IcebergObjectMapper.createIcebergObjectMapper(IcebergObjectMapper.java:25)
	at com.datastrato.gravitino.catalog.lakehouse.iceberg.web.IcebergObjectMapper.<clinit>(IcebergObjectMapper.java:16)
	at com.datastrato.gravitino.catalog.lakehouse.iceberg.web.metrics.IcebergMetricsFormatter.<init>(IcebergMetricsFormatter.java:18)
	at com.datastrato.gravitino.catalog.lakehouse.iceberg.web.metrics.IcebergMetricsManager.<init>(IcebergMetricsManager.java:50)
	at com.datastrato.gravitino.catalog.lakehouse.iceberg.IcebergRESTService.initServer(IcebergRESTService.java:56)
	at com.datastrato.gravitino.catalog.lakehouse.iceberg.IcebergRESTService.serviceInit(IcebergRESTService.java:80)
```

Reply:
so here introducing these two modules to fix the problem.

Reply:
ok